index.html

```diff
@@ -0,0 +1,106 @@
+
+        </h1>
+
+        <div class="bg-light row">
+
+        <figure class="col-8 py-3 mx-auto">
+
+            <img src="images\Raja_Aziz_Bhatti.jpeg" alt="a soldier in a #1 dress" id="image">
+
+            <figcaption class="text-center">
+
+sodier wearing a number one dress.
+
+            </figcaption>
+
+        </figure>
+
+       </div>
+
+       </div>
+
+   
+
+        <div id="tribute-info" class="container">
+
+           <div class="row">
+
+               <div class="col-4"></div>
+
+               <div class="col-5 ">
+
+                <h5 class="py-2">Here's a time line of Major Aziz Bhatti Shaheed</h5>
+
+                <ul>
+
+                    <li>Raja Aziz Ahmed Bhatti was born in British Hong Kong on <strong>6 August 1928</strong> into a Punjabi Rajput family </li>
+                    <li>His father, Mohammad Abdullah Bhatti, was an alumnus of the Queen's College in Hong Kong who later served as an Inspector in the Hong Kong Police Force.</li>
+                    <li>Aziz Bhatti was educated in Hong Kong where he completed his matriculation and attended the Queen's College in Hong Kong but his education was halted due to the Japanese invasion and occupation of Hong Kong in <strong>1941</strong>.</li>
+                    <li>The Pakistan Army has paid homage to Major Raja Aziz Bhatti Shaheed who embraced martyrdom while defending the country during the <strong>1965</strong> war with India.</li>
+                    <li>He was drafted in the Imperial Japanese Navy in <strong>1944</strong>, first serving at the rank of the seaman recruit and as the tower watchman (observation post) before being directed to attend the officer school offered by the Imperial Japanese Navy due to his educational qualifications.</li>
+                    <li>He said that Major Aziz's gallant actions and exemplary leadership inflicted heavy losses on Indian forces, successfully repulsing enemy attack on Lahore.</li>
+
+                    <li>Every year, the anniversary is celebrated with traditional fervour and the valour of the Nishan-e-Haider awardee.</li>
+
+                    <li>Nishan-e-Haider is conferred upon soldiers who perform acts of great heroism in circumstances of extreme danger, or to those who demonstrate bravery of the highest order or devotion to the country in the presence of the enemy.</li>
+
+
+                    <br><br>
+
+                    <p><cite>Rouge on the face of <strong>shahadat</strong>, pride of the country and the nation are these fearless warriors, a strike of their sword wipes out the mightiest of foes this one who came out victorious in the struggle for the cause of ALLAH is lying here in the delight of the afterlife dream. Major Bhatti fought valiantly on Lahore Front, and is posthumously presented with the <strong>Nishan-e-Haider</strong>.
+ 
+
+                       --Shaheed Major Aziz Bhatti </cite></p>
+
+               </div>
+
+           </div>
+
+          
+
+            </ul>
```

Comment:
You can't have any other tags in the ul apart from the li tag I kindly suggest that you wrap the tags inside a li element. So as to follow the semantic elements.

Reply:
Nice one, you rock 🔥.